entrypoint.sh

```diff
@@ -1,49 +1,98 @@
 #!/usr/bin/env bash
-set -x
+#set -x
+declare -i statuscode
+DATE=$(date +%Y%m%d)
+checksize=1023 # this parameter is using for checking slowlog per hour by size. If the file's size is
+               # less than 1024 bytes the slowlog file isn't valid for parsing.
 
-# Start download slowlog from RDS instances in aws account
-    DATE=$(date +%Y%m%d)
+function info () {
+    if [[ ${DEBUGLEVEL} -gt 0 ]] ; then
+    echo "$@"
+    fi
+}
 
-    function downloadLog () {
-      local instanceID=$1
-      local log=$2
 
```

Comment:
# this function is wraper on aws rds download-db-log-file-portion
# parameters: $1 RDS DBInstanceIdentifier
#                        $2 slow_lof file name
#                        $3 local fie with slow log